node/cn/backend.go

```diff
@@ -480,12 +488,12 @@ func (s *CN) APIs() []rpc.API {
 		}, {
 			Namespace: "klay",
 			Version:   "1.0",
-			Service:   downloader.NewPublicDownloaderAPI(s.protocolManager.Downloader(), s.eventMux),
+			Service:   publicDownloaderAPI,
 			Public:    true,
 		}, {
-			Namespace: "klay",
+			Namespace: "eth",
 			Version:   "1.0",
-			Service:   filters.NewPublicFilterAPI(s.APIBackend, false),
+			Service:   publicFilterAPI,
```

Comment:
```suggestion
			Service:   publicDownloaderAPI,
```
@jimni1222 
Currently, only the method `Syncing` is duplicated(one in PublicDownloaderAPI which is subscription method and the other in `PublicKlayAPI` which is callback method), but we need to think about whether a similar case may occur in the future.

If so, we should separate all the structures as separately like below.
```go
type EthereumFilterAPI struct {
	publicFilterAPI   *filters.PublicFilterAPI
}
type EthereumPublicBlockChainAPI struct {
	publicBlockChainAPI      *PublicBlockChainAPI
}
// ...
```

I'm not sure which one is the more maintainable design.
cc. @aidan-kwon

Reply:
for now, i modified wrong service name.
And in my opinion, we can just do at one struct for now (except for downloader api), and if duplication is happened in the future, and then we can separate the structure.